app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,33 @@
+/* universal */
+body {
+  padding-top: 60px;
+}
+section {
+  overflow: auto;
+}
```

each block should have a white line 
```suggestion
body {
  padding-top: 60px;
}

section {
  overflow: auto;
}
```

---

Sorry i am not sure what is the difference may I know how to do it?

---

Nevermind i understand now, I will fix